app/Role.php

```diff
@@ -17,6 +17,7 @@ class Role extends Model
     const DIRECTOR = 'director';
     const PRESIDENT = 'president';
     const STAFF = 'staff';
+    const CAMEL_BREEDER = 'camel-breeder';
```

Comment:
Note that this role has to be migrated if it's not a fresh migration. (ie. in production we already have roles and won't reload every migration)